test/e2e/apigatewayv2/resources/import_api.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apigatewayv2.services.k8s.aws/v1alpha1
+kind: API
+metadata:
+  name: "$API_NAME"
+spec:
+  body: '{
+            "openapi": "3.0.1",
+            "info": {
+              "title": "$API_TITLE",
+              "version": "v1"
+            },
+            "paths": {
+              "/": {
+                "get": {
+                  "x-amazon-apigateway-integration": {
+                    "uri": "http://example.com",
+                    "httpMethod": "GET",
+                    "type": "HTTP_PROXY",
+                    "payloadFormatVersion": "1.0"
```

Comment:
Not sure if it matters, but the payloadFormatVersion for the authorizer above is 2.0, and this is 1.0?

Reply:
It does not matter. 
For integrations, only supported version is 1.0 and this field was introduced to evolve in future.
However for authorizers format version "2.0" is simplified response format and we use it in tests. : )